modules/kafka-connect/src/main/clojure/xtdb/kafka/connect/util.clj

```diff
@@ -0,0 +1,21 @@
+(ns xtdb.kafka.connect.util
+  (:import (org.apache.kafka.connect.connector ConnectRecord)))
+
+(defn clone-connect-record [^ConnectRecord record {:keys [topic
```

Comment:
please use the indentation practices of the rest of the project.

```clojure
(defn clone-connect-record [^ConnectRecord record,
                            {:keys [topic kafka-partition
                                    key-schema key value-schema value
                                    timestamp headers]}]
  ...)
```

We minimise this so that we don't have to scroll as much to see related code.